examples/dt/nova/nova02beta/service-values.yaml

```diff
@@ -0,0 +1,58 @@
+# local-config: referenced, but not emitted by kustomize
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  preserveJobs: false
+  neutron:
+    customServiceConfig: |
+      [ml2]
+      mechanism_drivers = ovn,sriovnicswitch
+      [ml2_type_vlan]
+      network_vlan_ranges = sriov-phy4
+  ovn:
+    ovnController:
+      nicMappings:
+        datacentre: ospbr
+  glance:
+    customServiceConfig: |
+      [DEFAULT]
+      enabled_backends = default_backend:swift
+      [glance_store]
+      default_backend = default_backend
+      [default_backend]
+      swift_store_create_container_on_put = True
+      swift_store_auth_version = 3
+      swift_store_auth_address = {{ .KeystoneInternalURL }}
+      swift_store_endpoint_type = internalURL
+      swift_store_user = service:glance
+      swift_store_key = {{ .ServicePassword }}
+    default:
+      replicas: 1
+  swift:
+    enabled: true
+  nova:
+    apiServiceTemplate:
+      customServiceConfig: |
+        [pci]
+        alias = { "vendor_id":"8086", "product_id":"154d", "device_type":"type-PCI", "name":"a1" }
```

Comment:
from upstream docs https://docs.openstack.org/nova/latest/admin/pci-passthrough.html#pci-tracking-in-placement we must also enable pci.report_in_placement